### PR TITLE
cache get_torque_params

### PR DESCRIPTION
**Pull request by cl0cks4fe**
lots of io in get_torque_params (and it's is called twice in get_params for a lot of cars)

could also pass it through
re: https://github.com/commaai/openpilot/issues/32536

I'll look for further ways to speed up

### CODE REVIEW

File: selfdrive/car/interfaces.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from enum import StrEnum
 from typing import Any, NamedTuple
 from collections.abc import Callable
+from functools import lru_cache
```
Comment (sshane) [inline selfdrive/car/interfaces.py:9]: just use cache?

**Comment by sshane** [comment]
nice! speeds up the test with 8 procs from ~37 to ~30s for me! I wonder why the unit tests take 2x longer in CI though

**Comment by sshane** [comment]
Can you see why the unit tests take much longer now?

**Comment by cl0cks4fe** [comment]
@sshane selfdrive unittests?
Looks like it's ~1m faster than [this run](https://github.com/commaai/openpilot/actions/runs/9312796667/job/25634155524).

this PR: `1491 passed, 364 skipped, 4010 subtests passed in 420.02s (0:07:00)`
that one: `1491 passed, 364 skipped, 4015 subtests passed in 477.50s (0:07:57)`

**Comment by sshane** [comment]
The run on master is only 2m. Looks like we have some bug causing some runs to go for longer than usual. https://github.com/commaai/openpilot/actions/runs/9312989919/job/25634659494

You or @deanlee can feel free to look into it

**Comment by cl0cks4fe** [comment]
@sshane 
[different runners](https://github.com/commaai/openpilot/actions/runs/9305681411/workflow#L125-L127), which explains why your PRs are also fast. is this not intended?

File: selfdrive/car/interfaces.py [hunk 2]
```diff
@@ -42,7 +43,7 @@ class LatControlInputs(NamedTuple):
 
 TorqueFromLateralAccelCallbackType = Callable[[LatControlInputs, car.CarParams.LateralTorqueTuning, float, float, bool, bool], float]
 
-
+@cache
```
Comment (sshane) [inline selfdrive/car/interfaces.py:46]:
```suggestion

@cache
```

**Comment by deanlee** [comment]
https://github.com/commaai/openpilot/pull/32586 is a further optimization.

**Pull request merged**